generators/app/templates/src/FOOBAR/package.json

```diff
@@ -6,63 +6,66 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --prod && del-cli ./wwwroot/index.html && move-cli ./wwwroot/manifest.webmanifest ./wwwroot/manifest.json",
-    "build-dev": "ng build --source-map && del-cli ./wwwroot/index.html && move-cli ./wwwroot/manifest.webmanifest ./wwwroot/manifest.json",
-    "swagger-generate": "del-cli ./ClientApp/app/shared/models/api && mkdirp ./ClientApp/app/shared/models/api && swagger-angular-generator --no-store -s ../swagger/spec.json -d ./ClientApp/app/shared/models/api && for directory in common controllers store; do del-cli ./ClientApp/app/shared/models/api/${directory}; done",
-    "swagger-compare": "swagger-diff ../swagger/oldSpec.json ../swagger/newSpec.json",
+    "build:dev": "ng build --source-map && del-cli ./wwwroot/index.html && move-cli ./wwwroot/manifest.webmanifest ./wwwroot/manifest.json",
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "compodoc": "./node_modules/.bin/compodoc -p ClientApp/tsconfig.app.json -n \"TITLEFOO documentation\""
+    "generate": "swagger-angular-generator --no-store -s ../swagger/spec.json -d ClientApp/app/models/api",
+    "swagger:generate": "del-cli ClientApp/app/models/api && mkdirp ./ClientApp/app/shared/models/api && swagger-angular-generator --no-store -s ../swagger/spec.json -d ./ClientApp/app/shared/models/api && for directory in common controllers store; do del-cli ./ClientApp/app/shared/models/api/${directory}; done",
+    "compodoc:generate": "./node_modules/.bin/compodoc -p ClientApp/tsconfig.app.json -n \"LEZ documentation\"",
+    "compodoc:serve": "./node_modules/.bin/compodoc -s ClientApp/tsconfig.app.json"
   },
   "private": true,
   "dependencies": {
-    "@acpaas-ui/ngx-components": "3.3.0",
-    "@angular/animations": "9.0.2",
-    "@angular/cdk": "9.0.0",
-    "@angular/common": "9.0.2",
-    "@angular/compiler": "9.0.2",
-    "@angular/core": "9.0.2",
-    "@angular/forms": "9.0.2",
-    "@angular/platform-browser": "9.0.2",
-    "@angular/platform-browser-dynamic": "9.0.2",
-    "@angular/platform-server": "9.0.2",
-    "@angular/router": "9.0.2",
-    "@angular/service-worker": "9.0.2",
+    "@acpaas-ui/ngx-forms": "^4.2.2",
+    "@acpaas-ui/ngx-layout": "^4.2.1",
+    "@acpaas-ui/ngx-pagination": "^4.2.1",
+    "@acpaas-ui/ngx-table": "^4.5.1",
+    "@angular/animations": "10.1.6",
+    "@angular/cdk": "10.2.5",
+    "@angular/common": "11.0.0-next.4",
```

Comment:
Is er een reden om hier de next versie i.p.v. 10.x te gebruiken?

Reply:
De Angular updates gebeuren via de CLI (ng update) waarbij Angular zelf de verschillende versies bepaald voor de onderliggende componenten. Ik gebruik gewoon de stable releases bij ng update, dus eigenlijk geen idee waarom er een -next versie gebruikt wordt.